### PR TITLE
Convert to typescript

### PR DESCRIPTION
**Pull request by avalynndev**
Idk what you wanna do. delete files if you like..

### CODE REVIEW

**Comment by avalynndev** [comment]
## To publish the package:

- Delete LIB & Typings folder
- Run in terminal:
    - `npx tsc`
    - `npm publish`
- **REMEMBER:** change the package version in `package.json`

File: index.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+// Modules
```
Comment (wdhdev) [inline index.ts]: Can you alphabetise this section

File: package.json [hunk 2]
```diff
@@ -1,27 +1,37 @@
...
+		"test": "node test",
+		"build": "tsc",
+		"git": "sh ./gitpush.sh",
+		"dev": "npm publish --tag dev"
+	},
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/EasyScriptJS/EasyScript.git"
+	},
+	"keywords": [
```
Comment (wdhdev) [inline package.json:17]: Please update this section with the latest keywords in main branch.

File: package.json [hunk 3]
```diff
@@ -1,27 +1,37 @@
...
+		"type-converter",
+		"types",
+		"uuid"
+	],
+	"author": "Easy Script <packages@easyscript.dev>",
+	"bugs": {
+		"url": "https://github.com/EasyScriptJS/EasyScript/issues"
+	},
+	"homepage": "https://easyscript.dev",
+	"license": "CC BY-NC-ND 4.0",
```
Comment (wdhdev) [inline package.json]:
```suggestion
	"license": "MIT",
```

**Review by wdhdev — CHANGES_REQUESTED** [review]
Please update all code to the latest in main.

File: package.json (hunk 2 above)
Comment (avalynndev) [reply package.json:17]: ok

Comment (avalynndev) [reply package.json:17]: fixed

File: index.ts (hunk 1 above)
Comment (avalynndev) [reply index.ts]: Done

**Pull request closed without merging**